src/schema/v2/sales.ts

```diff
@@ -16,13 +21,13 @@ export const SalesConnectionField: GraphQLFieldConfig<void, ResolverContext> = {
     //       this was meant for refetching purposes, then we should add a plural
     //       `nodes` root field and use that instead.
     //
-    // ids: {
-    //   type: new GraphQLList(GraphQLString),
-    //   description: `
-    //     Only return sales matching specified ids.
-    //     Accepts list of ids.
-    //   `,
-    // },
+    id: {
```

Comment:
Also needed this 😅. I noticed it was removed from v2. I added it back and made the slight adjustment to use `id` instead of `ids`.

Reply:
Updated.